torchinductor/ir.py

```diff
@@ -1911,11 +1911,22 @@ def __init__(self, layout, inputs, constant_args=()):
             and inputs[0].get_device().type == "cuda"
         ):
             self.kernel = "triton_ops_conv"
+        elif (
+            config.autotune
```

Comment:
if both `config.triton.use_conv` and `config.autotune` are set we will ignore `config.autotune`

I'd suggest
```
config.triton.convolution = "aten"
or
config.triton.convolution = "triton"
or
config.triton.convolution = "autotune"
```